cucumber/authenticators_jwt/features/authn_jwt_check_standard_claims.feature

```diff
@@ -0,0 +1,367 @@
+    Then host "myapp" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: jwks-uri configured with correct value, issuer configured with correct value, iss claim with correct value, 200 OK
+    Given I have a "variable" resource called "test-variable"
+    And I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: issuer
+    """
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "issuer" variable to value "http://jwks"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject",
+      "iss": "http://jwks"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with raw service ID
+    Then host "myapp" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: jwks-uri configured with correct value, issuer configured with wrong value, iss claim with correct value, 401 Error
+    Given I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: issuer
+    """
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "issuer" variable to value "incorrect.com"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject",
+      "iss": "http://jwks"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    Invalid issuer. Expected incorrect.com, received http://jwks
+    """
+
+  Scenario: jwks-uri configured with wrong value, issuer configured with correct value, iss claim with correct value, 401 Error
+    Given I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: issuer
+    """
+    And I successfully set authn-jwt "jwks-uri" variable to value "incorrect.com"
+    And I successfully set authn-jwt "issuer" variable to value "incorrect.com"
```

Comment:
you wrote `issuer configured with correct value` so its confusing

Reply:
Fixed to incoreect